### PR TITLE
Adding DSTU2 documentation for Binary

### PR DESCRIPTION
**Pull request by zsura**

### CODE REVIEW

File: content/dstu2/binary.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+* TOC
+{:toc}
+
+## Retrieve by id
+
+List an individual Binary by its id:
+
+    GET /Binary/:id
+
+### Implementation Notes
```
Comment (MaxPhilips) [inline content/dstu2/binary.md]: In other resources, the implementation notes aren't a heading.  I'd change this to one italic line of "Implementation Notes - Binary for DiagnosticReport"

**Comment by davischristi** [comment]
Update conformance metadata with the binary resource.

File: content/dstu2/binary.md (hunk 1 above)
Comment (zsura) [inline content/dstu2/binary.md]: Done.

**Comment by zsura** [comment]
Updated conformance. There were a few other changes as well that required updating.
See https://github.com/cerner/fhir.cerner.com/pull/96/commits/f7c6a367d31dcb012aa456b768f2b36068cfb8b4

**Comment by davischristi** [comment]
Verified running it locally.
👍

**Comment by MaxPhilips** [comment]
+1

**Comment by cravelight** [comment]
Juxly Timeline is now using this endpoint as documented.  #shipit :)

**Comment by zsura** [comment]
Squashed commits. Merging.

**Pull request merged**